server/src/main/java/io/spine/server/model/Attribute.java

```diff
@@ -20,32 +20,24 @@
 package io.spine.server.model;
 
 import com.google.errorprone.annotations.Immutable;
-import io.spine.annotation.Internal;
 
 /**
- * Meta-data set to a {@link HandlerMethod#attributes()} HandlerMethod} attributes.
+ * Contains a value of an annotation parameter.
  *
  * <p>Typical way to add more semantics to a method is via a parameterized annotation,
  * such as {@link io.spine.core.Subscribe Subscribe}.
+ *
+ * @param <V>
+ *         the type of the value of the attribute. As it would typically be an annotation field
+ *         value, its type cannot be restricted to anything except for {@link Object}.
+ * @see HandlerMethod#attributes()
  */
-@Internal
 @Immutable(containerOf = "V")
-public interface MethodAttribute<V> {
+public interface Attribute<V> {
```

Comment:
Maybe, it's a `Parameter`. The doc hints on that.

Also, I don't understand why the method is called `parameter` and not `name`.